flood_ml/data_utils.py

```diff
@@ -0,0 +1,64 @@
+"""Data functions for Flood CNN model."""
+
+from constants import GEO_FEATURES, MAP_HEIGHT, MAP_WIDTH, N_FLOOD_MAPS
```

Comment:
Google style guide sez

"Use import statements for packages and modules only, not for individual types, classes, or functions."

https://google.github.io/styleguide/pyguide.html#22-imports

which mirrors

```
>>> import this
The Zen of Python, by Tim Peters

...
Namespaces are one honking great idea -- let's do more of those!
```

Reply:
Fixed.